### PR TITLE
Feature/rs 8539 add key flag to screener questions assisting

### PR DESCRIPTION
**Pull request by PLUCS2**
For RS #8539 issue

### CODE REVIEW

File: lib/generators/surveyor/templates/db/migrate/add_is_key_column_to_questions.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+class AddIsKeyColumnToQuestions < ActiveRecord::Migration[5.2]
+  def self.up
+    add_column :questions, :is_key, :boolean, default: false, null: false
+  end
+
+  def self.down
+    remove_column :questions, :is_key
+  end
+end
```
Comment (bsaris-ut) [inline lib/generators/surveyor/templates/db/migrate/add_is_key_column_to_questions.rb:12]:
```suggestion
end

```
I think your IDE should take care of this.

**Review by bsaris-ut — APPROVED** [review]

**Comment by bsaris-ut** [comment]
Also, should we be adding a version bump here or do we think that'd be it's own PR?

File: lib/generators/surveyor/templates/db/migrate/add_is_key_column_to_questions.rb (hunk 1 above)
Comment (michaeldewolf85) [reply lib/generators/surveyor/templates/db/migrate/add_is_key_column_to_questions.rb:12]: I'll fix this in my follow on for version bumps.

Comment (michaeldewolf85) [reply lib/generators/surveyor/templates/db/migrate/add_is_key_column_to_questions.rb:12]: I'll fix in my follow on for version bumps

**Review by michaeldewolf85 — APPROVED** [review]

**Pull request merged**